test/plots/aapl-bollinger.js

```diff
@@ -0,0 +1,20 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const AAPL = await d3.csv("data/aapl.csv", d3.autoType);
+  return Plot.plot({
+    y: {
+      grid: true
+    },
+    marks: [
+      Plot.areaY(AAPL, Plot.map({y1: bollinger(20, -2), y2: bollinger(20, 2)}, {x: "Date", y: "Close", fillOpacity: 0.2})),
+      Plot.line(AAPL, Plot.map({y: bollinger(20, 0)}, {x: "Date", y: "Close", stroke: "blue"})),
+      Plot.line(AAPL, {x: "Date", y: "Close", strokeWidth: 1})
+    ]
+  });
+}
+
+function bollinger(N, K) {
+  return Plot.window({k: N, reduce: Y => d3.mean(Y) + K * d3.deviation(Y)});
```

Comment:
```suggestion
  return Plot.window({k: N, reduce: Y => d3.mean(Y) + K * d3.deviation(Y), anchor: "end"});
```

I think BB are a trailing moving-average?

Also I see this note in wikipedia:
> Since Bollinger Bands use the population method of calculating standard deviation, the proper divisor for the sigma calculation is n, not n − 1, which would mean:

```suggestion
  return Plot.window({k: N, reduce: Y => d3.mean(Y) + K * d3.deviation(Y) * Math.sqrt(N/(N-1)), anchor: "end"});
```

(I don't think we need that much detail tho?)

Reply:
The second point is fascinating but I think I’m just going to ignore it for now.